.gitignore

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Please remove the gitignore. We maintain a simple readme file, hence no need of it.